Dockerfile

```
@@ -86,8 +88,7 @@ WORKDIR /app
 ENV PIP_DISABLE_PIP_VERSION_CHECK=1
 ENV PIP_NO_CACHE_DIR=1
 
-# rollback pip version to avoid legacy resolver problem
-RUN pip install pip==20.2.4;
+RUN pip install pip==23.1.2;
```

Is the updated pip version here important?

---

Older versions of pip will silently suppress conflict messages.

see https://github.com/getredash/redash/actions/runs/5491313994/jobs/10007730439?pr=5865#step:3:1473

---

Interesting.  So that's what the comment above is about. :smile: